.changelog/v1.0.0/summary.md

```diff
@@ -0,0 +1,52 @@
+*August 22nd, 2022*
+
+After more than 2 years in the works, this is the first stable release of the Hermes relayer! 🎉
+
+### Note for operators
+
+> ⚠️  If upgrading from Hermes v0.15.0, be aware that this release contains multiple breaking
+> ⚠️  changes to the Hermes command-line interface and configuration.
+> ⚠️  Please consult the [UPGRADING document for instructions](UPGRADING.md) ) for a more detadetails.
```

Comment:
I think `changelog.md` is a compilation of all the files in `.changelog/v1..`, including the summary. See https://github.com/informalsystems/unclog.

(BTW I think it might be useful if you get a bit more familiar with the release process Sean in case this is of interest to you. I think we have a recording somewhere of Romain and Shoaib doing a release.)